File: Week1/Day4/ExerciseXP_Gold/Exercise2.py
# Write a function that accepts one parameter (an int: X) and returns the value of X+XX+XXX+XXXX.
# Example:
# If X=3, the output when calling our function should be 3702 (3 + 33 + 333 + 3333)

# Hint: treating our number as a int or a str at different points in our code may be helpful

def func (x):
    my_list = [str(x)*i for i in [1, 2, 3, 4]]
    result = 0
    for j in my_list:
        result += int(j)
    return result

x = 3
print (func (x))

File: Week4/Day7/ExerciseXP/Exercise2.js
// Write a javascript function that takes an array of numbers and a target number.
// The function should find two different numbers in the array that, when added together, give the target number.
// For example: answer([1,2,3], 4) should return [1,3]

function answer(list, target) {
    for (i in list) {
        for (j = Number(i) + 1; j < list.length; j++) {
            if (list[i] + list[j] === target) {
                return [list[i], list[j]]
            }
        }
    }
}

a = [1,2,3,5,7];
x = 4;
console.log(answer(a, x));